ooni/report/tool.py

```diff
@@ -47,9 +47,9 @@ def upload(report_file, collector=None, bouncer=None):
     oonib_reporter = OONIBReporter(report.header, collector)
     log.msg("Creating report for %s with %s" % (report_file, collector))
     report_id = yield oonib_reporter.createReport()
+    report.header['report_id'] = report_id
```

Comment:
This is again to prefer the report identifier selected by the backend as opposed to using the one written inside the report (if any), correct?